daemon/emer-daemon.c

```diff
@@ -1726,6 +1746,21 @@ emer_daemon_class_init (EmerDaemonClass *klass)
                          G_PARAM_CONSTRUCT_ONLY | G_PARAM_READWRITE |
                          G_PARAM_STATIC_STRINGS);
 
+  /*
+   * EmerDaemon:aggregate-tally:
```

Comment:
```suggestion
   * EmerDaemon:aggregate-tally: (nullable)
```

Reply:
It can be `NULL` at construct time, but will always be non-`NULL` on a constructed object. Is this the right annotation for that case? (Maybe that cannot be expressed.)

Reply:
https://gi.readthedocs.io/en/latest/annotations/giannotations.html#type-signature suggests there is no way to express this.

Reply:
errrrrr, I actually meant to type `(not nullable)`, but got distracted by having to write out the GitHub suggestion syntax manually. Sorry.

`(not nullable)` is a valid annotation.

Reply:
No, I was right. I did intend to type `(nullable)` because you can pass `NULL` in at construction time.